src/assets.js

```diff
@@ -0,0 +1,2 @@
+import './styles/iconic/css//open-iconic-bootstrap.min.css';
+import './index.css';
```

Comment:
each file should end with an empty line from [https://github.com/airbnb/javascript#whitespace--newline-at-end](url)